### PR TITLE
Add 16 bit jpeg output mode to benchmark_xl.

### PR DESCRIPTION
**Pull request by szabadka**

### CODE REVIEW

File: tools/cjxl_main.cc [hunk 1]
```diff
@@ -584,7 +584,7 @@ jxl::Status GetPixeldata(const std::vector<uint8_t>& image_data,
 #endif
 #if JPEGXL_ENABLE_JPEG
     if (jxl::extras::DecodeImageJPG(encoded, color_hints, size_constraints,
-                                    &ppf)) {
+                                    /*output_bit_depth=*/8, &ppf)) {
```
Comment (jonsneyers) [inline tools/cjxl_main.cc:587]: Why not 16 here?

Comment (szabadka) [reply tools/cjxl_main.cc]: Right, it can be 16 now. I just forgot that I made it work with the old library in the meantime.

**Review by jonsneyers — APPROVED** [review]

**Comment by jyrkialakuijala** [comment]
could you post the results of the benchmark in the reviewlog -- people can have interest on how fast it is

**Review by jyrkialakuijala — COMMENTED** [review]
nice.

**Comment by szabadka** [comment]
> could you post the results of the benchmark in the reviewlog -- people can have interest on how fast it is

I edited the PR description to include benchmark results. There is about a 4% speed difference between 8-bit and 16-bit output.

**Pull request merged**